src/api/transformers/test/transfromTestToDTO.ts

```diff
@@ -3,15 +3,19 @@ import { TestDTO } from '../../interface';
 import { transformTestTypeToDTO } from './transformTestTypeToDTO';
 import { transformTestResultsToDTO } from './transformTestResultsToDTO';
 import { transformTestInterpretationToDTO } from './transformTestInterpretationToDTO';
+import { transformTestTypeToSimpleDTO } from './transformTestTypeToSimpleDTO';
 
 export function transformTestToDTO(test: Test): TestDTO {
   return {
     id: test.id.value,
     userId: test.userId.value,
     creationTime: test.creationTime,
+    administeredBy: {
+      userId: test.administeredBy.value,
```

Comment:
Why did you go nested for this one?  If we're going to nest, it makes me wonder if the administration confidence should be nested together as an administration section?

Reply:
I went for nested as we might need to add name information there (Results Entered By, Administered by) We can move the confidence in there too.
Something like:
```
administration: {
   userId: ``,
   confidence: ``,
}
```

And same for results:

```
entry: {
    userId: ``
    confidence: ``
}
```